### PR TITLE
feat: SSR-161 SQFormsDropdown - Update component

### PR DESCRIPTION
**Pull request by asim-sqfe**
Update the current implementation of the Dropdown with added specifications. Here is the current implementation of this SQForm: https://master--5f4431386ea00a00220d495c.chromatic.com/?path=/story/components-sqformdropdown--default (this is an example link)

Design reference: https://www.figma.com/file/DsJr2cFZqMYkOX6VjcuhqW/SC3-DESIGN-SYSTEM?node-id=1518%3A11678 
Typography: https://www.figma.com/file/DsJr2cFZqMYkOX6VjcuhqW/SC3-DESIGN-SYSTEM?node-id=340%3A9946 
Color: https://www.figma.com/file/DsJr2cFZqMYkOX6VjcuhqW/SC3-DESIGN-SYSTEM?node-id=340%3A9646 

Default:
Label:  Text Style: Label 12 px Weight: 400
Value: Text Style: Value 14px Weight: 600 Semi-bold

Focused:
Label: Text Style: Label 12 px Weight: 400
Value: Text Style: Value 14px Weight: 600 Semi-bold

With Validation:
Label: Text Style: Label 12 px Weight: 400
Value: Text Style: Value 14px Weight: 600 Semi-bold
Line: #C67000 Warning 
Helper Text/icon: #C67000 Warning   #3C7107 Text/Success 
Icon Size: 12 x 11 Warning  9 x 11 Success badge

Loom demo: [here](https://www.loom.com/share/b65e6120904b4768818229646869e4b2l)

### CODE REVIEW

**Comment by laurelbean** [comment]
Can you add a screenshot or loom showing the updated component?

File: src/components/fields/SQFormDropdown/SQFormDropdown.tsx [hunk 1]
```diff
@@ -137,7 +137,7 @@ function SQFormDropdown({
           {label}
         </InputLabel>
         <Select
-          sx={classes.selectHeight}
+          sx={(classes.selectHeight, {fontWeight: 600})}
```
Comment (aharpt) [inline src/components/fields/SQFormDropdown/SQFormDropdown.tsx:140]: Can we put the `fontWeight` into the `classes.selectHeight` class?

Comment (asim-sqfe) [reply src/components/fields/SQFormDropdown/SQFormDropdown.tsx]: done, thanks @aharpt

**Comment by asim-sqfe** [comment]
> Can you add a screenshot or loom showing the updated component?

loom link is added in the PR descriptions, please review

**Review by aharpt — APPROVED** [review]

**Review by laurelbean — APPROVED** [review]

**Pull request merged**